migrations/20180907154321_recreate_views_with_new_naming_conventions.js

```diff
@@ -0,0 +1,66 @@
+exports.up = async function(knex, Promise) {
+  await knex.raw(`DROP VIEW "PagesView"`);

```

Comment:
Do we need to drop the view given we are using `CREATE OR REPLACE VIEW`?

Reply:
That's what I thought but for some weird reason, if I try to use `create or replace` without dropping the views first I get an error in the migration preventing it from happening. It just complains saying I can't rename the columns...

Reply:
That's daft 😞